lib/govuk_publishing_components/presenters/related_navigation_helper.rb

```diff
@@ -142,26 +142,15 @@ def related_taxons
         @related_taxons ||= content_item_links_for("taxons", only: "taxon")
       end
 
-      def related_topics_or_taxons
-        return related_topics if related_topics.any?
+      def related_browse_topics_or_taxons
+        return related_browse_topics if related_browse_topics.any?
```

Comment:
Maybe something like the following would be better here?
```ruby
related_browse_topics.presence || related_taxons.presence || []
```

Reply:
Nice suggestion, thanks I'll use that 👍